### PR TITLE
Update _childetx.html.erb

### PR DESCRIPTION
**Pull request by childetx**
Fixed img link

### CODE REVIEW

**Review by moveson — DISMISSED** [review]

File: app/views/pages/_childetx.html.erb [hunk 1]
```diff
@@ -1,6 +1,8 @@
 <div class="card text-center">
-  <img src="https://photos.app.goo.gl/d5UNHeQvkEXDe6AF9"/>
+  <img src="https://lh3.googleusercontent.com/9SSkEO3PrPgs5TdTnnlIPC7COYSMOxTRkjFnDHN6HeZop3_Y9UVyVEvsXFy5kyEJtxrILEHJbOLaDUzA4EpWLQ5GgJ4PWO9tJ9GzbKJtMsy2cbQRf82cNezqLymOzZfSDIDYdo7u00bo1gL_1dL63SMRVWnx5sWuEiS6g_-hY4gfOS5_x6SX7iD1el5rNzUwJ9E9M9ai3bFMHFnrUTbEcCOntfNLc6RB4U51KI5Ab0CxP_zH8a7gMAgXby1Z3VrpunqOUKHL0TmKxrUhzk3X68RkzXgOpYY2bwicU8n1WfH-aUM2-KrhbdHzj3WYZkCJKy_KMdgv3yEWGvqjmOc8WeevPkIgSszP0azrcscA1daq2RBmloSbn-rG32tQx2S8pr7RNlIS52B6Am2plR2vq6WWKFNrx2ojSYRZ_bROCv8P42d_Fc8YWpbSFauO4BmxmSp2Knop1c6RSTGyDn10JoN-eLKOcI55FD-VcGKV7tadedCn6oPgKGn50jFFg1NSzAoEKM8jIJjDMx_KEP6wNH4cQ1llu4z9-XWj4KbXOZ6tlTfWUYgn7rvvxHs-M6eGRSAsk7vKztgWHqoss0sE9DlTTp4cVo5B1_dB9rg2obbYa0FQVG3EI9c2zk00fc5ep1-RJGptksMvL52gfMO_sINC3HHHg64rqzRBeRP8XXGDWfosadVmiGBn2txqWw=w934-h1245-no?authuser=0"/>
   <div class="card-body">
     <h4 class="card-title">The Child Fam</h4>
+    <p>This is our fam on the night that Brigham received his mission call!</p>
+    <p>He is going to Child Rancagua!</p>
```
Comment (DtheTchyld) [inline app/views/pages/_childetx.html.erb]: Do your spelling and grammer stuffs bruh

**Review by DtheTchyld — CHANGES_REQUESTED** [review]

File: app/views/pages/_childetx.html.erb (hunk 1 above)
Comment (moveson) [reply app/views/pages/_childetx.html.erb]: Yeah all the spelling things have to be spelled

Comment (childetx) [reply app/views/pages/_childetx.html.erb]: Spelling updated, bruh!

**Review by DtheTchyld — APPROVED** [review]
You fixed it

**Pull request merged**